PowerShell/ScubaGear/Modules/Providers/ExportAADProvider.psm1

```diff
@@ -89,6 +89,10 @@ function Export-AADProvider {
     }
     $ServicePlans = ConvertTo-Json -Depth 3 @($ServicePlans)
 
+    #Obtains license information for tenant and total number of active users
+    $LicenseInfo = $Tracker.TryCommand("Get-MgBetaSubscribedSku") | Select-Object -Property Sku*, ConsumedUnits, PrepaidUnits | ConvertTo-Json -Depth 3
+    $UserCount = Get-MgBetaUserCount -ConsistencyLevel eventual
```

Comment:
I initially tried to wrap in a Tracker.TryCommand, but when I was adding the consistency level it would cause the line to error out, and if I didn't include the consistency level then ScubaGear would prompt the user for the consistency level in the middle of the run. As a result I didn't wrap the line in the Tracker.TryCommand. If there is another way to do this and wrap it within the Tracker, please let me know and I can update it accordingly.